tests/p-chain-tests.ts

```diff
@@ -381,6 +381,80 @@ describe('Staking tests', async function () {
     await ui.promptsPromise;
   });
 
+  it('can sign a create subnet transaction', async function () {
+    const txn = Buffer.from([
+      0x00, 0x00,
+      0x00, 0x00, 0x00, 0x10, 0x00, 0x00, 0x30, 0x39,
+      // blockchain ID
+      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+      // number of outputs
+      0x00, 0x00, 0x00, 0x01,
+      // output
+      ... localAssetId,
+      0x00, 0x00, 0x00, 0x07, 0x00, 0x00, 0x00, 0x00,
+      0xee, 0x5b, 0xe5, 0xc0, 0x00, 0x00, 0x00, 0x00,
+      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
+      0x00, 0x00, 0x00, 0x01, 0xda, 0x2b, 0xee, 0x01,
+      0xbe, 0x82, 0xec, 0xc0, 0x0c, 0x34, 0xf3, 0x61,
+      0xed, 0xa8, 0xeb, 0x30, 0xfb, 0x5a, 0x71, 0x5c,
+      // number of inputs
+      0x00, 0x00, 0x00, 0x01,
+      // input
+      0xdf, 0xaf, 0xbd, 0xf5, 0xc8, 0x1f, 0x63, 0x5c,
+      0x92, 0x57, 0x82, 0x4f, 0xf2, 0x1c, 0x8e, 0x3e,
+      0x6f, 0x7b, 0x63, 0x2a, 0xc3, 0x06, 0xe1, 0x14,
+      0x46, 0xee, 0x54, 0x0d, 0x34, 0x71, 0x1a, 0x15,
+      // addresses?
```

Comment:
what is the `?`?

Reply:
Not sure why the question mark is there, will get rid of it